07_errors/alextmz/types.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+type PuppyID uint64
+type PuppyVal int32
```

Comment:
Nit: This should be float type. Because prices are like 999.99 😁😁

Reply:
It's not a nit, it's a real bug man! :+1: